docs/WritingTests.asciidoc

```diff
@@ -1274,16 +1274,75 @@ well as define a custom schedule using the test variable +SCHEDULE+.
 Also test modules can be defined and overridden on-the-fly using a
 downloadable asset.
 
+===== EXCLUDE_MODULES
+
+If a job has the following schedule:
+
+- boot/boot_to_desktop
+- console/systemd_testsuite
+- console/docker
+
+The module console/docker can be excluded with:
+
+----
+openqa-clone-job --from https://openqa.opensuse.org --host https://openqa.opensuse.org 24 EXCLUDE_MODULES=docker
+----
+
+The schedule would be:
+
+- boot/boot_to_desktop
+- console/systemd_testsuite
+
+NOTE: Excluding modules that are not scheduled does not raise and error.
+
+===== INCLUDE_MODULES
+
+If a job has the following schedule:
+
+- boot/boot_to_desktop
+- console/systemd_testsuite
+- console/docker
+
+The module console/docker can be excluded with:
```

Comment:
I guess it would make sense to also use an example for scheduling only the included parts, not the ones you want to exclude, e.g. when you have a list of "boot, mod1, mod2, … mod42" you can select to schedule only boot and mod13 with "INCLUDE_MODULES=boot,mod13"

Reply:
I think this can be confusing, because even if it is named "INCLUDE_MODULES", it is actually excluding the rest (as a white list).

I think that with this example people can understand the behavior better. The case when "you want to only schedule some parts" can be inference from these examples.

WDYT?